dali/imgcodec/decoders/libtiff/tiff_libtiff.cc

```diff
@@ -141,11 +141,12 @@ TiffInfo GetTiffInfo(TIFF *tiffptr) {
 
 }  // namespace detail
 
-DecodeResult LibTiffDecoderInstance::Decode(SampleView<CPUBackend> out, ImageSource *in,
```

Comment:
This decoder was not parallelized due to implementing Decode instead of DecodeImplTask